### ISSUE TITLE
PR #214 breaks backward compatibility

### ISSUE THREAD

**Issue by the-ress** [lduchosal/ipnetwork#220]
PR #214 added an optional parameter to `IPNetwork.Parse` method.

This breaks other libraries calling `IPNetwork.Parse` with 1 parameter causing them to fail with `MissingMethodException` (unless they're recompiled against the new IPNetwork version).

Here's more info about why this happens: https://stackoverflow.com/a/9884700/99237

A backward compatible approach should look like this:
```csharp
public static IPNetwork Parse(string network)
{
    return Parse(network, true);
}

public static IPNetwork Parse(string network, bool sanitize)
{
   // ...
}
```

Also I'd like to point out a typo in the current parameter name: `sanitanize` -> `sanitize`.

**Comment by the-ress** [comment]
A concrete example of an affected library: [richardschneider/net-mdns](https://github.com/richardschneider/net-mdns):

```
System.TypeInitializationException: The type initializer for 'Makaretu.Dns.MulticastService' threw an exception.
 ---> System.MissingMethodException: Attempted to access a missing method.
   at Makaretu.Dns.MulticastService..cctor()
   --- End of inner exception stack trace ---
   at Makaretu.Dns.MulticastService..ctor(Func`2 filter)
```

**Comment by Ch4mundi** [comment]
Same for me :-(

**Comment by the-ress** [comment]
Here's PR with the fix: #221
